src/prisma/schema.prisma

```diff
@@ -0,0 +1,27 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url = env("DATABASE_URL")
+}
+
+model User {
+  id Int @id @default(autoincrement())
+  email String @unique
+  password String
+  firstName String?
+  lastName String?
+  products Product[]
+}
+
+model Product {
+  id Int @id @default(autoincrement())
+  name String
+  size Float
+  description String?
+  price Int
+  owner User? @relation(fields: [ownerId], references: [id])
+  ownerId Int?
```

Comment:
Follow the ``product-registration``, It should contains color, category and delivery option
```suggestion
  id Int @id @default(autoincrement())
  name String
  color String
  size Float
  description String?
  price Int
  owner User? @relation(fields: [ownerId], references: [id])
  ownerId Int?
  category String
  deliveryOption String
```